projects/planner/src/app/plan/diveoptions/diveoptions.component.spec.ts

```diff
@@ -52,5 +61,25 @@ describe('Dive options component', () => {
         fixture.detectChanges();
         component.isComplex = false;
         expect(spy).toHaveBeenCalledWith(false);
-    }));
+    }
+    ));
+
+    it('should call switchAirBreaks on selectedOptions', () => {
+        const switchSpy = spyOn(schedules.selectedOptions, 'switchAirBreaks');
+        component.switchAirBreaks();
+        expect(switchSpy).toHaveBeenCalledWith();
```

Comment:
Add also verification to all tests, that the scheduler was called to recalculate dives.